provisioners/file/home/ec2-user/bin/limit-logs

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd /data/jenkins/jobs
+sudo find . -type f -name '*log' -size +15M \
+    -exec truncate --size 15M {} \;
```

Comment:
Supposedly `execdir` is safer, though I never quite understood why.

Reply:
Per find manpage avoids races in path resolution.
Changed with commit 5f4ac1ddbc5349bd9d607afb808001070b44de43